### PR TITLE
week-9-a

### PR DESCRIPTION
**Pull request by SriramKodey**
The equations are not getting rendered as I expected in Git Hub. Use VScode if possible. I will try to correct this with another pull request if possible.
--**Fixed**

### CODE REVIEW

File: bite-size-notes/week-9-b-Extented_Kalman_Filter.md
Comment (sdpattonUW) [inline bite-size-notes/week-9-b-Extented_Kalman_Filter.md:1]: The images don't seem to be rendering in the document when I try to display the rich diff of the file, it's just blue text linking (presumable what should be the image) to a 404 not found page. Am I viewing this wrong or is there an actual error?

File: bite-size-notes/week-9-b-Extented_Kalman_Filter.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  </div>
+</div>
+<div>
+  <div style="margin-right:10px;" align="center">
+    <img src="figs/EKF_2.png" alt="Non-Linear Transformation of Gaussian distribution" width="400" height="400" align="center"/>
+     <p align="center"><em>Figure 2: Non-Linear Transformation of Gaussian distribution</em></p>
+  </div>
+</div>
+
+## Preliminaries
```
Comment (sdpattonUW) [inline bite-size-notes/week-9-b-Extented_Kalman_Filter.md]: I really like that you include the review of these principles before going into their use in the EKF.

File: bite-size-notes/week-9-b-Extented_Kalman_Filter.md
Comment (sdpattonUW) [reply bite-size-notes/week-9-b-Extented_Kalman_Filter.md:1]: NVM, it seems it is an issue with the code being a pull request at the moment and should be resolved after being pushed into the main bite sized notes folder.

File: bite-size-notes/week-9-a-Extented_Kalman_Filter.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+      <em>Figure 6: Linearised Transformation of low uncertainity distribution</em>
+    </p>
+  </div>
+</div>
+
+- Thus, in practice, when applying EKFs it is important to keep the uncertainity of the state estimate small.
+- EKFs have been applied with great success to a number of state estimation problems that violate the underlying assumptions. 
+
+### EKF-Slam Example
+Simultaneous Localization and Mapping (SLAM) is a process where a robot or autonomous system builds a map of an unknown environment while simultaneously tracking its location within that map. SLAM is essential for autonomous navigation in complex and dynamic environments. Historically the earilest-and perhaps the most influential- SLAM algorithm is based on the extended Kalman filter. The figures attached showcase the implementaion of EKF Slam in a physics simulator with different motion and measurement uncertainities. Notice the drop in performance of the EKF with increase in uncertainities.
```
Comment (sdpattonUW) [inline bite-size-notes/week-9-a-Extented_Kalman_Filter.md]: Could you possibly explain what we're looking at in the image here? It's easy to see that the green circles are the estimation of the obstacles' positions but what is the red and blue line that snakes around the ground? Is that the trajectory of the robot?

Comment (SriramKodey) [reply bite-size-notes/week-9-a-Extented_Kalman_Filter.md]: Thanks for pointing that out. I have made the changes.

File: bite-size-notes/week-9-b-Extented_Kalman_Filter.md
Comment (SriramKodey) [reply bite-size-notes/week-9-b-Extented_Kalman_Filter.md:1]: Yes, once the changes are committed the images will be added to the main repo and can be sourced in the markdown file.

**Pull request merged**